dll/shellext/fontext/lang/de-DE.rc

```diff
@@ -0,0 +1,29 @@
+// Don't translate this file at current time
```

Comment:
:thinking:

Reply:
It's not the only translation where is information was ignored

Reply:
Ah if it's the same, then it's okay.

Reply:
So let's make it worse by also ignoring it.
Guess I'll either never update this shellext again, or ignore all languages.